vyper/cli/vyper_compile.py

```diff
@@ -64,6 +68,20 @@ def _cli_helper(f, output_formats, compiled):
         print(json.dumps(compiled), file=f)
         return
 
+    if output_formats == ("archive",):
+        for contract_data in compiled.values():
+            assert list(contract_data.keys()) == ["archive"]
+            out = contract_data["archive"]
+            if f.isatty() and isinstance(out, bytes):
+                raise RuntimeError(
+                    "won't write raw bytes to a tty! (if you want to base64"
+                    " encode the archive, you can try `-f archive` in"
+                    " conjunction with `--base64`)"
```

Comment:
wouldn't "you can try `f archive --base64`" be better?

Reply:
they don't necessarily need to go in that order